### ISSUE TITLE
Round app icon

### ISSUE THREAD

**Issue by simontb** [ligi/PassAndroid#236]
For Android 10 devices a dedicated round icon would be beneficial. It doesn't look really nice right now:

![Screenshot_20191027-114513](https://user-images.githubusercontent.com/6491719/67633365-4e694d80-f8af-11e9-914b-8e525f0ec633.png)

**Comment by ligi** [comment]
True. But I can't create icons and the round one has to be at least the quality the current one has. And tbh I am not a fan of the whole round icon thing ..

**Comment by simontb** [comment]
A round icon would be an addition, so user prefering other shapes could pick a launcher that displays the original icon.

Where does the original icon come from that you're unable to create a new one?

I just simply created a round icon from the Android Studio wizard and think it already looks better with the Pixel launcher (though screenshot is from emulator):

![Screenshot_1573736059](https://user-images.githubusercontent.com/6491719/68858836-46e6d880-06e6-11ea-93ab-8b95e37cdf40.png)

If you want I could work on a PR. But if it wouldn't be merged because of general objections against alternative shapes, I'd restrain from the work.

**Comment by frozenspider** [comment]
Bumping this up - on ASUS Zenfone's version of Android 9 there's no way to change icon shape (it has been deliberately removed), so I couldn't change change round shapes even if I wanted to.
